config/zones/SE.yaml

```diff
@@ -157,6 +161,14 @@ emissionFactors:
       - datetime: '2021-01-01'
         source: electricityMap, 2021 average
         value: 42.07081808517848
+    nuclear:
+      datetime: '2020-01-01'
+      source: UNECE, https://unece.org/sites/default/files/2022-04/LCA_3_FINAL%20March%202022.pdf#page=56
+      value: 5.13
+    solar:
+      datetime: '2021-01-01'
+      source: INCER ACV, https://docs.google.com/spreadsheets/d/1w5DJ7sPen6axIHU8TCVcuzNCjlct4I6JAbhUlw-ZXu8/edit?usp=sharing
+      value: 35.76666667
```

Comment:
I know this is a draft but these seems to be missing from the zone level SE zones, is this just an oversight?

At least for nuclear it should be pretty straight forward as SE-SE3 is the only zone which has nuclear power plants right now.

Reply:
We've only added the configuration at zone level. That's a good point, I'll add the overrides for the country levels as well.

Reply:
This is at the county level, but it's missing from the zone level. 😅
This is probably due to the fact that SE used to be a zone level zone as well.